app/views/docs/configuration.phtml

```diff
@@ -168,6 +172,29 @@
     </tr>
     </thead>
     <tbody>
+    <tr>
+        <td>_APP_DOMAIN</td>
+        <td>
+            Your Appwrite domain address. 
+            When setting a public suffix domain, Appwrite will attempt to issue a valid SSL certificate automatically. 
+            When used with a dev domain, Appwrite will assign a self-signed SSL certificate.
+            If you're using a proxy for <b>localhost development</b>, such as <a href="https://ngrok.com/" target="_blank" rel="noopener">ngrok</a>, this will be the domain of your localhost proxy.
+        </td>
+    </tr>
+    <tr>
+        <td>_APP_DOMAIN_TARGET</td>
+        <td>
+            A DNS A-record hostname to serve as a CNAME target for your Appwrite custom domains. 
+            You can use the same value as used for the Appwrite <code>_APP_DOMAIN</code> variable.
+            If you're using a proxy for <b>localhost development</b>, such as <a href="https://ngrok.com/" target="_blank" rel="noopener">ngrok</a>, this will be the domain of your localhost proxy, such as <code>dd65-2405-201-4013-d8d7-b4c5-fb73-39f9-285c.ngrok.io</code>.
+        </td>
+    </tr>
+    <tr>
+        <td>_APP_DOMAIN_FUNCTIONS</td>
+        <td>
+            [@steven @khushboo help]
```

Comment:
Something like:

```suggestion
            This will be used for <a href="/docs/functions-deploy#domains">Function Domains</a>. When a function domain is generated, it will be <code>[UNIQUE_ID].[_APP_DOMAIN_FUNCTIONS]</code>.
            You can use the same value as used for the Appwrite <code>_APP_DOMAIN</code> variable.
```